pkgs/by-name/fy/fyne/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+
+  dbus,
+  libGL,
+  libX11,
+  libXcursor,
+  libXinerama,
+  libXi,
+  libXrandr,
+  libXxf86vm,
+  pkg-config,
+}:
+
+let
+  version = "2.4.4";
+in buildGoModule {
+  pname = "fyne";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "fyne-io";
+    repo = "fyne";
+    rev = "v${version}";
+    hash = "sha256-6gDHURcf4dVmaiXleeRKogAtCAgcHU437TjEHhfGdj8=";
+  };
+
+  vendorHash = "sha256-0P2YqXq+c20F2hVkRcvwV5PpqUoZ3PnRIvTHt/qAMPA=";
+
+  nativeBuildInputs = [
+    pkg-config
+  ];
+
+  buildInputs = [
+    libGL
+    libX11
+    libXcursor
+    libXinerama
+    libXi
+    libXrandr
+    libXxf86vm
+  ];
+
+  checkInputs = [
+    dbus
+  ];
+
+  doCheck = true;
+
+  preCheck = ''
+    OLD_DIR=$PWD
```

Comment:
OLD_DIR and OLD_GOPATH seems a bit strange

Reply:
OLD_GOPATH was unnecessary, but OLD_DIR is needed to get tests to succeed.